teams.go

```diff
@@ -6,60 +6,7 @@ import (
 )
 
 type TeamsResponse struct {
-	Data []struct {
-		ID              int    `json:"id"`
-		DivisionID      int    `json:"divisionId"`
-		Handle          string `json:"handle"`
-		Name            string `json:"name"`
-		AbbreviatedName string `json:"abbreviatedName"`
-		Logo            struct {
-			Main struct {
-				Svg string `json:"svg"`
-				Png string `json:"png"`
-			} `json:"main"`
-			MainName struct {
-				Svg string `json:"svg"`
-				Png string `json:"png"`
-			} `json:"mainName"`
-		} `json:"logo"`
-		HasFallback bool   `json:"hasFallback"`
-		Location    string `json:"location"`
-		Players     []struct {
-			ID       int    `json:"id"`
-			Handle   string `json:"handle"`
-			Name     string `json:"name"`
-			FullName string `json:"fullName"`
-			Role     string `json:"role"`
-			Accounts []struct {
-				ID   int    `json:"id"`
-				Type string `json:"type"`
-				URL  string `json:"url"`
-			} `json:"accounts"`
-			Number       FlexibleInteger `json:"number"` // Empty is "" ?!? use custom unmarshaler to handle
```

Comment:
I think some of these fields got dropped in the new player struct, the problem with this api is a lot of the structs they should be pretty similar between api calls aren't.

Reply:
Yeah, it's a bit of an annoying API to work with, agreed. Adding JSON samples to your tests would help greatly.